lib/supa/commands/attributes.rb

```diff
@@ -0,0 +1,23 @@
+module Supa
+  module Commands
+    class Attributes < Supa::Command
+      def represent
+        names.each do |name|
+          Supa::Commands::Attribute.new(
+            @subject, representer: @representer, tree: @tree, name: name, options: attribute_options
```

Comment:
Just pass them, `attribute` will decide how to work with them. It's not a responsibility of `attributes`. For instance then it will be possible to define single modifier to all and so on. And when we will need to add new one, we will not whitelist it